roles/slurm/tasks/build.yml

```diff
@@ -127,6 +134,11 @@
       path: "{{ slurm_build_dir }}"
       state: absent
 
+  - name: remove old lib dir
```

Comment:
This should only be executed when `slurm_build` is true. Otherwise this makes the Slurm role non-idempotent, and could break functional clusters.

Reply:
> This should only be executed when `slurm_build` is true. Otherwise this makes the Slurm role non-idempotent, and could break functional clusters.

It is already under the same block with `when: slurm_build` with other item above it.